packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -17,9 +17,9 @@ This guide is an overview of building your own user interface using the embedded
 
 ## Sign-in flow
 
-Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of *authenticators*, or ways of verifying the identity of a user, and assigns them to apps, people, groups, and more. Policies also configure whether an authenticator is required or optional, as well as the minimum number of authenticators required for a successful sign-on. Many flows require multiple *factors* (multifactor authentication), the category of an authenticator. Factors include biometrics, such as a fingerprint, knowledge, such as a password, and more. This results in a many of possible combinations of the different authenticators, the steps in the sign-in flow.
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of *authenticators*, or ways of verifying the identity of a user, and assigns them to apps, people, groups, and more. Policies also configure whether an authenticator is required or optional, as well as the minimum number of authenticators required for a successful sign-in. Responding to each authenticator is a step in the sign-in flow. Many flows require multiple *factors* (multifactor authentication), the category of an authenticator. Factors include: biometrics, such as a fingerprint; knowledge, such as a password; and more. This effectively results in an infinite number of combinations of authenticators and the order in which they're presented.
 
-The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta org app integration, request the initial step in the flow, and cycle through responding to steps until either the user signs in, cancels, or an error occurs.
+The Android and Swift Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of the Okta org application integration, request the initial step in the flow, and cycle through responding to steps until either the user signs in, cancels, or an error occurs.
```

Comment:
```suggestion
The Android and Swift Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of an Okta org application integration, request the initial step in the flow, and cycle through responding to steps until either the user signs in, cancels, or an error occurs.
```

Reply:
Change made. It will be in the next commit